patches/video.js+7.13.3.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/node_modules/video.js/dist/video.es.js b/node_modules/video.js/dist/video.es.js
+index a07aa77..3f7bc30 100644
+--- a/node_modules/video.js/dist/video.es.js
++++ b/node_modules/video.js/dist/video.es.js
+@@ -13959,7 +13959,7 @@ var ProgressControl = /*#__PURE__*/function (_Component) {
+     }
+ 
+     var seekBarEl = seekBar.el();
+-    var seekBarRect = findPosition(seekBarEl);
++    var seekBarRect = getBoundingClientRect(seekBarEl);
```

Comment:
This should fix the video.js bug, where the timeline tooltip is getting out of bounds on the right. Found it here https://github.com/videojs/video.js/pull/7308